NebulaHost/PacketProcessors/Factory/Entity/CreatePrebuildsRequestProcessor.cs

```diff
@@ -131,7 +130,7 @@ public void ProcessPacket(CreatePrebuildsRequest packet, NebulaConnection conn)
                 //Check only, if buildPreview has some connection to another prebuild
                 if (preview.coverObjId < 0)
                 {
-                    tmpVector = pab.previewPose.position + pab.previewPose.rotation * preview.lpos;
+                    tmpVector = preview.lpos;
```

Comment:
This is how it was changed in the game itself but testing needs to be done to make sure this still works as intended